docs/develop/add-ins-with-angular2.md

```diff
@@ -71,27 +71,24 @@ const routes: Routes = // route definitions go here
 export class AppRoutingModule { }
 ```
 
-
 ## Consider wrapping Fabric components with Angular components
 
-We recommend using [UI Fabric](https://developer.microsoft.com/fabric#) styling in your add-in. UI Fabric for the web is available in two flavors: 
+We recommend using [UI Fabric](https://developer.microsoft.com/fabric#) styling in your add-in. UI Fabric for the web is available in two flavors:
 
 - [Fabric React](https://developer.microsoft.com/fabric#/controls/web) provides robust, up-to-date, accessible components that are highly customizable.
 
 - [Fabric Core](https://developer.microsoft.com/fabric#/styles/web) is a collection of CSS classes and Sass mixins that give you access to Fabric's colors, animations, fonts, icons and grid.
 
-Consider using Fabric components in your add-in by wrapping them in Angular components. For an example that shows you how to do this, see [Word Style Checking Add-in Built on Angular](https://github.com/OfficeDev/Word-Add-in-Angular2-StyleChecker). Note, for example, how the Angular component defined in [fabric.textfield.wrapper](https://github.com/OfficeDev/Word-Add-in-Angular2-StyleChecker/blob/master/app/shared/office-fabric-component-wrappers/fabric.textfield.wrapper.component.ts) imports the Fabric file TextField.ts, where the Fabric component is defined. 
-
+Consider using Fabric components in your add-in by wrapping them in Angular components. For an example that shows you how to do this, see [Word Style Checking Add-in Built on Angular](https://github.com/OfficeDev/Word-Add-in-Angular2-StyleChecker). Note, for example, how the Angular component defined in [fabric.textfield.wrapper](https://github.com/OfficeDev/Word-Add-in-Angular2-StyleChecker/blob/master/app/shared/office-fabric-component-wrappers/fabric.textfield.wrapper.component.ts) imports the Fabric file TextField.ts, where the Fabric component is defined.
 
 ## Using the Office dialog API with Angular
 
-The Office Add-in dialog API enables your add-in to open a page in a nonmodal dialog box that can exchange information with the main page, which is typically in a task pane.
+The Office Add-in dialog API enables your add-in to open a page in a non-modal dialog box that can exchange information with the main page, which is typically in a task pane.
```

Comment:
I think "nonmodal" is correct. English doesn't put a hyphen after a prefix except in special circumstances when necessary to avoid confusion.

Reply:
Ricky is correct! Plus, it's in the styleguide:
**non- (Microsoft Writing Style Guide)**
Don't hyphenate words beginning with non-, such as nonnumeric and nonzero, unless a hyphen is necessary to avoid confusion, as in non-native, or non- is followed by a proper noun, as in non-English. When in doubt, check [The American Heritage Dictionary](https://ahdictionary.com/) or consult your group's editorial style guide.

Don't use non- to negate an entire phrase.

_Example_
unrelated to security (instead of non-security related)
To learn more about hyphenation, see [Hyphens](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28765).